README.md

```diff
@@ -24,25 +24,16 @@ licence and version as Guile)
 Additional Contributors
 -------------------------------------------------------------------------------
 Kai-Martin Knaak
-
+James Larrowe
 
 How to use
 -------------------------------------------------------------------------------
 
 By calling:
 
-    $ ./build-commit v2.0.9
```

Comment:
You removed the example of how to build a specific commit. Let people know the argument to ./build-commit is an actual git hash or git tag

Reply:
Have you tried this? Does it work?